src/agent/communicator/src/communicator.cpp

```diff
@@ -47,6 +46,9 @@ namespace communicator
             {
                 LogInfo("Using insecure connection.");
             }
+
+            std::string retryIntervalSecs = getStringConfigValue("agent", "retry_interval_secs");
```

Comment:
Maybe you can simplify these two lines into one.

Reply:
Done